recipes/help2man/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "help2man" %}
+{% set version = "1.47.4" %}
+{% set sha256 = "d4ecf697d13f14dd1a78c5995f06459bff706fd1ce593d1c02d81667c0207753" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{name}}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - automake
+    - perl
+  run:
+    - perl
+
+test:
+  commands:
+    - help2man --help
+    - test -f ${PREFIX}/bin/help2man
+
+about:
+  home: https://www.gnu.org/software/help2man/
+  license: GPL-3.0
+  license_family: GPL
```

Comment:
I don't think this works with `conda-build` 1, but it will work with `conda-build` 2. For 1 IIRC this needs to be GPL3.